pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -42,6 +43,10 @@ const (
 	// diskrecorder would refuse to write files larger than MaxLogSize, so GatherClusterOperators
 	// has to limit the expected size of the buffer for logs
 	logCompressionRatio = 2
+
+	// imageGatherPodLimit is the maximum number of pods that
+	// will be listed in a single request to reduce memory usage.
+	imageGatherPodLimit = 64
```

Comment:
Would it be possible to measure what will be the memory consumption compared to for example 500 pods batch ? I would like to limit it, but also too small batch size would prolong download times.
One idea how to do it would be to add same pod 500 times to array and sleep for a long time to get memory size.
Also, what would be the archive size in this case (with 500 items) ?
In my understanding, [this](http://superset-dh-prod-superset.cloud.datahub.psi.redhat.com/superset/dashboard/OCP_etcd_object_counts/) indicates that we might expect about 2500 pods in a cluster.